packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -270,6 +282,24 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
     );
   }
 
+  Widget _buildStatusLine(BuildContext context) {
+    final inset = DevToolsScaffold.borderInsets.bottom;
+
+    return Container(
+      height: 48.0,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: <Widget>[
+          const PaddedDivider(padding: EdgeInsets.all(0.0)),
```

Comment:
you should be able to use EdgeInsets.zero for this

Reply:
changed